templates/web/pages/product_edit/product_edit_form_packagings.tt.html

```diff
@@ -0,0 +1,182 @@
+<!-- start templates/[% template.name %] -->
+
+<initjs>
+var packaging_shapes_select2_options = [% generate_select2_options_for_taxonomy_to_json("packaging_shapes") %];
+var packaging_materials_select2_options = [% generate_select2_options_for_taxonomy_to_json("packaging_materials") %];
+var packaging_recycling_select2_options = [% generate_select2_options_for_taxonomy_to_json("packaging_recycling") %];
+
+var component = 0;
```

Comment:
what is this for ?

Reply:
I don't remember, it's not used, I guess I changed my mind and forgot it.